.ci/release-settings.xml

```diff
@@ -0,0 +1,29 @@
+<settings>
+  <servers>
+    <server>
+      <id>sonatype-nexus-staging</id>
+      <username>SONATYPE_USER</username>
+      <password>SONATYPE_PWD</password>
+    </server>
+    <server>
+      <id>sourceforge-new-site</id>
+      <username>SOURCEFORGE_USER</username>
+      <password>SOURCEFORGE_PWD</password>
+    </server>
+  </servers>
+
+  <profiles>
+    <profile>
+      <!-- "mvn release:prepare .... -Pgpg"
+           and it is possible to skip " -Dgpg.passphrase=xxxx" at "-Darguments"
+           Use "gpg \-\-list-keys", "pub   1024D/C6EED57A 2010-01-13"
+                to get "gpg.keyname" in example it is  value "C6EED57A"
```

Comment:
please remove things that causes issues with spellchecker - xxxx and EED

Reply:
Also, consider moving this to some wiki page / readme

Reply:
I believe we can move this to, contribution repository and creating a new ReadME file there.

Reply:
content is from https://github.com/checkstyle/checkstyle/wiki/How-to-make-a-release

@Rahulkhinchi03 , please use "....." instead of "xxxx".

I placed this comment a while ago, as signal on how to keep

Reply:
Changes Done